#!/bin/bash
#SBATCH --mem-per-cpu=4gb
#SBATCH --time=12:00:00
#SBATCH --job-name=ABBAc
#SBATCH --error=ABBAc
#SBATCH --output=ABBAc
#SBATCH --partition=mpi
#SBATCH --ntasks=96

module load python2


indID=$((SLURM_ARRAY_TASK_ID -1))
inName=(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21)


#phy_himN_favW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_favW_her; fi

#phy_himN_favE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_favE_her; fi

#phy_himS_favW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_favW_her; fi

#phy_himS_favE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_favE_her; fi







#phy_himN_emmW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_emmW_her; fi

#phy_himN_emmE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_emmE_her; fi

#phy_himS_emmW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_emmW_her; fi

#phy_himS_emmE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_emmE_her; fi




#phy_himN_cyrN_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_cyrN_her; fi

#phy_himN_cyrS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_cyrS_her; fi

#phy_himS_cyrN_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_cyrN_her; fi

#phy_himS_cyrS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_cyrS_her; fi





#himN_himS_favW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_favW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638,BC2639 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_favW_her; fi

#himN_himS_favE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2565,BC2566,BC2567,BC2570,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favE BC2565,BC2566,BC2567,BC2570 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_favE_her; fi


#himN_himS_emmW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_emmW_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmW BC2563,BC2577,BC2578,BC2579 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_emmW_her; fi

#himN_himS_emmE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_emmE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_emmE_her; fi

#himN_himS_cyrN_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_cyrN_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_cyrN_her; fi


#himN_himS_cyrS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_himS_himN_cyrS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s BC2565,BC2566,BC2567,BC2570,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 himS BC2565,BC2566,BC2567,BC2570 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_himS_himN_cyrS_her; fi










#phy_himS_favW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himS_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himS_favWs_her; fi

#phy_himN_favW_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_himN_favWs_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,himera_001,himera_002,himera_003,himera_006,himera_030,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 favW BC2635,BC2637,BC2638 -P2 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_himN_favWs_her; fi

#ven_west_east_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_ven_west_east_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 5 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s M_3654,M_3655,M_3656,M_3657,M_3659,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr_east_west.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 5 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P3 east BC2563,BC2577,BC2578,BC2579,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 west cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -P1 ven M_3654,M_3655,M_3656,M_3657,M_3659 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_ven_west_east_her; fi





#phy_emmW_himS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_emmW_himS_her; fi

#phy_favW_himS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_phy_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,BC2639,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favW BC2635,BC2637,BC2638,BC2639 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 phy STRI_WOM_5732,STRI_WOM_5742,STRI_WOM_5765,STRI_WOM_5766 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_phy_favW_himS_her; fi


#emmE_emmW_himS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_emmE_emmW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2565,BC2566,BC2567,BC2570,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 emmW BC2563,BC2577,BC2578,BC2579 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_emmE_emmW_himS_her; fi


#favE_favW_himS_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_favE_favW_himS_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2565,BC2566,BC2567,BC2570,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 favWs BC2635,BC2637,BC2638 -P3 himS BC2565,BC2566,BC2567,BC2570 -P1 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_favE_favW_himS_her; fi


#cyrN_cyrS_himN_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 24 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 100000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_cyrN_cyrS_himN_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861,himera_001,himera_002,himera_003,himera_006,himera_030,cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P2 cyrS cyrbia_004,cyrbia_005,cyrbia_023,cyrbia_024 -P3 himN himera_001,himera_002,himera_003,himera_006,himera_030 -P1 cyrN cyrCAM040545,cyrCAM040578,cyrCAM040584,cyrCAM040673,cyrCAM040692,cyrCAM040695,cyrCAM040842,cyrCAM040853,cyrCAM040860,cyrCAM040861 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_cyrN_cyrS_himN_her; fi


#emmW_emmE_favE_her
#if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato01_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato02_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato03_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato04_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato05_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato06_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato07_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato08_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato09_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato10_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato11_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato12_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato13_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato14_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato15_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato16_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato17_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 48 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 100000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato19_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato20_emmW_emmE_favE_her; fi
#if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 2 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato21_emmW_emmE_favE_her; fi

#if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 96 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 100000 -s GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,BC2563,BC2577,BC2578,BC2579,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 emmW BC2563,BC2577,BC2578,BC2579 -P3 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -P2 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_emmW_emmE_favE_her; fi

if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 96 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat phased --podSize 2000 -s GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555,GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675,BC2635,BC2637,BC2638,hermathena_13,hermathena_14,hermathena_15 --popsFile ~/work/scripts/Popmap_erato_cyr.txt | python ~/work/scripts/genomics_general-master/ABBABABAwindows.py -T 2 --windType coordinate -w 50000 -s 50000 -m 250 --minData 0.5 -f phased -P1 favWs BC2635,BC2637,BC2638 -P3 emmE GS020redo,GS021redo,NCS_1671,NCS_1672,NCS_1673,NCS_1674,NCS_1675 -P2 favE GS012redo,NCS_0471,NCS_0473,NCS_0476,NCS_0478,NCS_0479,NCS_2554,NCS_2555 -O her hermathena_13,hermathena_14,hermathena_15 -o ~/work/ABBABABA_new/Herato18_favW_favE_emmE_her; fi
