#!/bin/bash
#SBATCH --mem-per-cpu=8gb
#SBATCH --time=72:00:00
#SBATCH --job-name=calls2
#SBATCH --error=calls2
#SBATCH --output=calls2
#SBATCH --partition=mpi
#SBATCH --ntasks=1

module load python2
#module load jdk/11.0.7

indID=$((SLURM_ARRAY_TASK_ID -1))
inName=(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21)


if [ "$(echo "${inName[indID]}")" == 1 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato1.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr1_demophoon -c Herato0101; fi
if [ "$(echo "${inName[indID]}")" == 2 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato2.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr2_demophoon -c Herato0201,Herato0202,Herato0203,Herato0204,Herato0205,Herato0206,Herato0207,Herato0208,Herato0209,Herato0210,Herato0211,Herato0212,Herato0213,Herato0214,Herato0215; fi
if [ "$(echo "${inName[indID]}")" == 3 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato3.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr3_demophoon -c Herato0301,Herato0302,Herato0303,Herato0304,Herato0305,Herato0306,Herato0307,Herato0308,Herato0309,Herato0310; fi
if [ "$(echo "${inName[indID]}")" == 4 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato4.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr4_demophoon -c Herato0401,Herato0402,Herato0403,Herato0404,Herato0405,Herato0406,Herato0407,Herato0408,Herato0409,Herato0410,Herato0411,Herato0412,Herato0413,Herato0414,Herato0415,Herato0416,Herato0417,Herato0418,Herato0419; fi
if [ "$(echo "${inName[indID]}")" == 5 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato5.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr5_demophoon -c Herato0501,Herato0502,Herato0503,Herato0504,Herato0505,Herato0506,Herato0507,Herato0508,Herato0509,Herato0510,Herato0511; fi
if [ "$(echo "${inName[indID]}")" == 6 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato6.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr6_demophoon -c Herato0601,Herato0602,Herato0603,Herato0604,Herato0605,Herato0606,Herato0607,Herato0608,Herato0609; fi
if [ "$(echo "${inName[indID]}")" == 7 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato7.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr7_demophoon -c Herato0701; fi
if [ "$(echo "${inName[indID]}")" == 8 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato8.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr8_demophoon -c Herato0801,Herato0802,Herato0803,Herato0804,Herato0805,Herato0806,Herato0807,Herato0808,Herato0809,Herato0810,Herato0811,Herato0812,Herato0813,Herato0814,Herato0815,Herato0816,Herato0817,Herato0818,Herato0819,Herato0820,Herato0821; fi
if [ "$(echo "${inName[indID]}")" == 9 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato9.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr9_demophoon -c Herato0901,Herato0902,Herato0903,Herato0904; fi
if [ "$(echo "${inName[indID]}")" == 10 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato10.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr10_demophoon -c Herato1001,Herato1002,Herato1003,Herato1004,Herato1005,Herato1006,Herato1007; fi
if [ "$(echo "${inName[indID]}")" == 11 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato11.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr11_demophoon -c Herato1101,Herato1102,Herato1103,Herato1104,Herato1105,Herato1106,Herato1107,Herato1108,Herato1109,Herato1110,Herato1111,Herato1112,Herato1113,Herato1114,Herato1115,Herato1116; fi
if [ "$(echo "${inName[indID]}")" == 12 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato12.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr12_demophoon -c Herato1201,Herato1202; fi
if [ "$(echo "${inName[indID]}")" == 13 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato13.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr13_demophoon -c Herato1301; fi
if [ "$(echo "${inName[indID]}")" == 14 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato14.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr14_demophoon -c Herato1401,Herato1402,Herato1403,Herato1404,Herato1405,Herato1406,Herato1407,Herato1408,Herato1409,Herato1410,Herato1411; fi
if [ "$(echo "${inName[indID]}")" == 15 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato15.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr15_demophoon -c Herato1501,Herato1502,Herato1503,Herato1504,Herato1505,Herato1506,Herato1507,Herato1508,Herato1509,Herato1510,Herato1511,Herato1512,Herato1513,Herato1514,Herato1515,Herato1516,Herato1517,Herato1518,Herato1519,Herato1520,Herato1521,Herato1522,Herato1523,Herato1524; fi
if [ "$(echo "${inName[indID]}")" == 16 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato16.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr16_demophoon -c Herato1601,Herato1602,Herato1603,Herato1604,Herato1605; fi
if [ "$(echo "${inName[indID]}")" == 17 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato17.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr17_demophoon -c Herato1701,Herato1702,Herato1703,Herato1704,Herato1705,Herato1706,Herato1707,Herato1708,Herato1709,Herato1710,Herato1711,Herato1712,Herato1713,Herato1714,Herato1715,Herato1716,Herato1717,Herato1718,Herato1719; fi
if [ "$(echo "${inName[indID]}")" == 18 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato18.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr18_demophoon -c Herato1801,Herato1802,Herato1803,Herato1804,Herato1805,Herato1806,Herato1807; fi
if [ "$(echo "${inName[indID]}")" == 19 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato19.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr19_demophoon -c Herato1901,Herato1902,Herato1903,Herato1904,Herato1905,Herato1906,Herato1907,Herato1908,Herato1909,Herato1910; fi
if [ "$(echo "${inName[indID]}")" == 20 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato20.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr20_demophoon -c Herato2001; fi
if [ "$(echo "${inName[indID]}")" == 21 ]; then python ~/work/scripts/genomics_general-master/filterGenotypes.py -t 10 -i /work/rpapa/share/calls_erato/chromosomes/Herato_040518.Herato21.calls.gz --inputGenoFormat phased --outputGenoFormat diplo | python sweepfinder_input.py -p "demophoon[Pet_ED3,Pet_ED4,Pet_ED5,Pet_ED6,STRI_WOM_0033,STRI_WOM_0082,STRI_WOM_0087,STRIWOM1284,STRIWOM5353,STRIWOM5362];hermathena[hermathena_13,hermathena_14,hermathena_15]" -O hermathena -o chr21_demophoon -c Herato2101; fi


